Dockerfile

```diff
@@ -154,6 +150,15 @@ RUN wget ${WGET_ARGS} https://static.rust-lang.org/rustup/rustup-init.sh && \
 	cargo install uefi-run --root /usr && \
 	rm -f ./rustup-init.sh
 
+# Install Zephyr SDK as 'user' in order to ensure that the `Zephyr-sdk` CMake
+# package is located in the package registry under the user's home directory.
+USER user
```

Comment:
Do you need to switch back to root or whatever user was active previously?

Reply:
I am no Docker expert, but does `USER` instruction persist?
(i.e. when we build `Dockerfile.user`, will we be running as `user` at the beginning? If so, we should indeed switch back to `root`)

Reply:
https://docs.docker.com/engine/reference/builder/#user

> The USER instruction sets the user name (or UID) and optionally the user group (or GID) to use when running the image and for any RUN, CMD and ENTRYPOINT instructions that follow it in the Dockerfile.

It looks like it persists. I will add `USER root` after this.